src/dom.js

```diff
@@ -239,6 +239,23 @@ export function lastChildElement(parent, callback) {
   return null;
 }
 
+/**
+ * Finds all child nodes that satisfies the callback.
+ * These nodes can include Text, Comment and other child nodes.
+ * @param {!Node} parent
+ * @param {function(!Node):boolean} callback
+ * @return {!Array.<!Node>}
+ */
+export function childNodes(parent, callback) {
```

Comment:
@erwinmombay The other similar functions (`childElement`, `childElements`, `lastChildElement`) in this file use `callback`. What's the difference between the word `callback` and `predicate`? `callback` can do anything inside but `predicate` will return true or false value?